pyproject.toml

```
@@ -41,9 +41,9 @@ notifications-python-client = "^6.3.0"
 help-desk-client = "==0.2.0"
 govuk-bank-holidays = "^0.11"
 detect-secrets = "^1.4.0"
-dit-activity-stream = "^0.1.0"
 django-workflow-engine = "0.2.0"
 govuk-frontend-django = "0.3.7"
+dit-activity-stream = "0.2.3"
```

Can we update this to
```suggestion
dit-activity-stream = "^0.2.3"
```